### PR TITLE
[better-allauth-logging] Log auth errors

### PR DESCRIPTION
**Pull request by wlonk**
zebra

### CODE REVIEW

**Comment by wlonk** [comment]
@jgerigmeyer I think that I have a solution.

I'm using a custom adapter so that I can log errors and get them into sentry. But it'll still land on the template that Allauth uses for login errors. So I'm copying that template into the repo, and we can change it to be the same as the index template that renders the React app, possibly with some flag set to say "you had a login error". I am happy to talk about this more if you have questions, I doubt I'm explaining it well.

File: config/settings/base.py [hunk 1]
```diff
@@ -314,10 +315,16 @@ def env(name, default=NoDefaultValue, type_=str):
 # Raven / Sentry
 SENTRY_DSN = env('SENTRY_DSN', default='')
 
-
+LOG_REQUEST_ID_HEADER = "HTTP_X_REQUEST_ID"
+LOG_REQUESTS = True
```
Comment (cdcarter) [inline config/settings/base.py]: can we set 
```
GENERATE_REQUEST_ID_IF_NOT_IN_HEADER = True
REQUEST_ID_RESPONSE_HEADER = "X-Request-ID"
```
as well?

**Comment by wlonk** [comment]
@jgerigmeyer Before we merge this, I'd love your input on the "don't show the raw allauth error page" approach I suggested, and perhaps to get that change in.

**Comment by jgerigmeyer** [comment]
@wlonk That works for me. Should I go ahead and modify the `authentication_error.html` template?

**Comment by wlonk** [comment]
@jgerigmeyer Please! I think that should do what we need, if you do. Do you want it to be just the same as `index.html`, or to include an error message variable or something?

**Comment by wlonk** [comment]
@jgerigmeyer 👍

**Pull request merged**